src/commands/sfpowerkit/org/sandbox/refresh.ts

```diff
@@ -131,4 +174,19 @@ export default class Refresh extends SFPowerkitCommand {
 
     return sandbox_query_result.records[0].Id;
   }
+
+  public async getSandboxDescription(conn: Connection, name: string) {
+    const query_uri = `${conn.instanceUrl}/services/data/v${this.flags.apiversion}/tooling/query?q=SELECT+Description+FROM+SandboxInfo+WHERE+SandboxName+in+('${name}')`;
+
+    const sandbox_query_result = await request({
+      method: "get",
+      url: query_uri,
```

Comment:
Thanks @Janani-26 , Would you be able to refactor this to use query executor?
https://github.com/Accenture/sfpowerkit/blob/main/src/utils/queryExecutor.ts

Reply:
@azlam-abdulsalam Refactored using Query Executor

Reply:
@azlam-abdulsalam - Could you please confirm if there are any other code refactoring has to be completed for this refresh command.